Slim/Utils/Misc.pm

```diff
@@ -634,6 +637,43 @@ sub getLibraryName {
 	return $hostname;
 }
 
+=head2 getTempDir()
+
+	Get LMS-private temp dir 
+
+=cut
+
+sub getTempDir {
+	return $tempdir;	
+}
+
+=head2 makeTempDir()
+
+	make and clean LMS-private temp dir 
+
+=cut
+
+sub makeTempDir {
+	$tempdir = catdir($prefs->get('cachedir'), 'tmp');
```

Comment:
I'd say return here early if `$tempdir` already is defined. As `makeTempDir()` now is easily available, it might invite people to accidentally call it more than once. But we don't want that folder to be wiped after the initialisation!

Reply:
done